projects/wireguard/README.md

```diff
@@ -15,14 +15,10 @@ A full technical paper from NDSS 2017 is available [here](https://www.wireguard.
 ## Contents
 
 ### Kernel Patches
-This project keeps Linux kernel patches for WireGuard against a 4.9.x kernel.  
-This kernel is built into the `mobylinux/kernel-wireguard` image that is generated by `cd kernel-wireguard && make`.
-
-WireGuard can also be included as a kernel module.
+The `kernel/` sub-directory contains a kernel patch that is included by the main kernel package of LinuxKit.
```

Comment:
I would move the docs to `docs/wireguard.md` - once we merge this we can remove the wireguard project directory, and and a note in projects/README.md saying it was merged.